app/src/main/java/controller/Controller.java

```diff
@@ -1,30 +1,40 @@
 package controller;
 
+
+import controller.customerController.CustomersController;
+import controller.elevatorSystemController.ElevatorSystemController;
 import lombok.Setter;
 import model.Model;
 import java.util.concurrent.TimeUnit;
 
+/**
+ * Main controller .
+ *
+ * @see CustomersController
+ * @see ElevatorSystemController
+ */
 public class Controller {
+    public final ElevatorSystemController ELEVATOR_SYSTEM_CONTROLLER;
+    private final CustomersController CUSTOMER_CONTROLLER;
     public final Model MODEL;
 
     private final int TPS = 50;
```

Comment:
Nie wiem czy bym nie zaimplementował ilości FPS względem wydajności danego komputera. Fajnie byłoby móc grać w symulator z 200 FPS jak ktoś ma dobry komputer

Reply:
Myślę, że to ciekawe, ale zbędne. Też nie wiem jak możnaby było to rozwiązać.